src/components/tabController/useScrollToItem.ts

```diff
@@ -100,8 +100,9 @@ const useScrollToItem = <T extends ScrollToSupportedViews>(props: ScrollToItemPr
     innerSpacing = 0
   } = props;
   const itemsWidths = useRef<(number | null)[]>(_.times(itemsCount, () => null));
-  const itemsWidthsAnimated = useSharedValue(_.times(itemsCount, () => 0));
-  const itemsOffsetsAnimated = useSharedValue(_.times(itemsCount, () => 0));
+  // Math.max(itemsCount, 2) is for setting minimum of 2 items required for the interpolation
+  const itemsWidthsAnimated = useSharedValue(_.times(Math.max(itemsCount, 2), () => 0));
+  const itemsOffsetsAnimated = useSharedValue(_.times(Math.max(itemsCount, 2), () => 0));
```

Comment:
This looks like a simple solution, but it feels technically incorrect since we're returning an array with a length of 2 instead of 1.
However, when testing for another solution it will probably be more code change than this.
WDYT?

Reply:
I preferred a simple fix. 
TBH, I initially thought about not supporting it all and instead throw an appropriate error to the users that they must pass at least two items. WDYT? 
I don't see a reason to use TabController for a single page..

Reply:
Sounds like a possible solution.
Is there an option in one of our apps where there can be a single tab? If it is possible, then maybe we should handle it and not force the users to have an if\else.

Reply:
I don't think it's relevant.. 
The person who reported the issue, just played with the component and passed a single item, but it wasn't his intention so it wasn't really a problem. 
So I think having an appropriate error message in this case will be suffice...

Reply:
Ok, I added a console.error instead of throwing and handled the issue a little better (I think)
Let me know what u think..